### DISCUSSION TITLE
Liquid Ajax Cart: Error while performing cart Ajax request

### DISCUSSION THREAD

**Discussion by javaidnaik** [EvgeniyMukhamedjanov/liquid-ajax-cart#22] [General]
Please check the error, getting this error on each page refresh
![shop_error](https://user-images.githubusercontent.com/68897797/167777395-f874b8f7-f886-40fb-8cff-69aaf3146d80.png)
.

**Comment by EvgeniyMukhamedjanov** [comment]
Hey @javaidnaik 
Can I have a preview link to check?
